capstone-server/src/main/java/com/hcl/capstoneserver/invoice/dto/InvoiceDTO.java

```diff
@@ -0,0 +1,112 @@
+package com.hcl.capstoneserver.invoice.dto;
+
+import com.hcl.capstoneserver.invoice.CurrencyType;
+import org.hibernate.validator.constraints.Currency;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.validation.constraints.NotBlank;
+
+public class InvoiceDTO {
+    private Integer invoiceId;
+    @NotBlank(message = "client id is required")
+    private String clientId;
```

Comment:
client id should not be an user input. we have to get it from the logged-in user